index.html

```diff
@@ -1,7 +1,7 @@
 <html>
 <h1> Welcome to Salem Sri Giri Travels</h1>
 <body>
-This website is under construction
+This website is under construction. Please remember to visit us later.
```

Comment:
How about changing to " We will be ready soon! bookmark this page now and remember to visit us soon" ?